src/hub.js

```diff
@@ -541,6 +541,15 @@ const onReady = async () => {
   const initialEnvironmentEl = document.createElement("a-entity");
   initialEnvironmentEl.addEventListener("bundleloaded", () => {
     remountUI({ initialEnvironmentLoaded: true });
+
+    for (const modelEl of initialEnvironmentEl.querySelectorAll(":scope > [gltf-model-plus]")) {
+      const model = modelEl.object3DMap.mesh;
+
+      if (model && model.animations.length > 0) {
+        modelEl.setAttribute("animation-mixer", "");
```

Comment:
I think this routine should be extracted into a plain utility function for now -- eg is there a reason we check for loop animation above but not here?

Reply:
I was thinking we probably don't want to automatically add the `loop-animation` component to the environment. They should probably be specified in the glTF model you are using as your environment. Idk, maybe we should use the same behavior? Would that mess any environments up?

Reply:
it seems like a good idea to make these consistent so we will naturally parameterize things properly as we hit edge cases -- I don't think any of our existing environments have one-shot animations so probably ok on that front -- any other concerns with adding the loop?

Reply:
Idk, let's turn it on and get feedback.